spec/routing/exhibitions_routing_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
```

Comment:
These looks like Scaffold specs that actually shouldn't be passing. We don't want update via PUT to work for example. But having the full resources route noted above would make them pass. 

The only routes that should be available are index and show, as everything else is handled by administrate.